docs/dg/dev/backend-development/data-manipulation/data-publishing/publish-and-synchronization.md

```diff
@@ -241,6 +241,40 @@ class SynchronizationBehaviorConfig extends SprykerSynchronizationBehaviorConfig
 </table>
 ```
 
+**Environment Limitations**
+
+**Disabled Dynamic Multi-Store (DMS) environment:**
+
+When using the DMS-OFF behavior, the Direct Sync feature has the following limitations:  
```

Comment:
When the **DMS** feature is turned off, the **Direct Sync** feature has the following limitations:
**Single-store configuration:** The feature is only supported for configurations with a single store.
**Multi-store configuration with namespace consistency:** For configurations with multiple stores, all stores must use the same Storage and Search namespaces."